pbiviz.json

```diff
@@ -2,14 +2,14 @@
   "visual": {
     "name": "drilldownCartogram",
     "displayName": "Drilldown cartogram",
-    "guid": "drilldownCartogramD5E1077C70074FFCB26AFC2A636614BF",
+    "guid": "drilldownCartogramD5E1077C70074FFCB26AFC2A636618BF",
```

Comment:
GUID shouldn't be changed if we want to deliver the new version to all of end users.

Reply:
@ignatvilesov I changed this for a reason:
The fix was working fine with PowereBI Service, but not when I package and use the visual in Desktop version. When I changed the "guid" it was working fine. Any suggestions?

Reply:
PBI loads custom visuals from our CDN. That means that PBI loads the published version of custom visual instead of loading from the package.
Anyway, we should keep the same GUID if there aren't breaking changes.

Reply:
Ok. I reverted the old GUID